ports/tensorflow-cc/portfile.cmake

```
@@ -75,51 +67,313 @@ set(ENV{TF_NEED_IGNITE} 0)
+			file(APPEND ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-${BUILD_TYPE}/.bazelrc "startup --host_jvm_args=-Djavax.net.ssl.trustStorePassword='$ENV{BAZEL_CUSTOM_CACERTS_PASSWORD}'\n")
+			message(STATUS "Using supplied custom CA certificate store password.")
+		endif()
+	else()
+		if(DEFINED ENV{HTTPS_PROXY})
+			message(STATUS "You are using HTTPS_PROXY. In case you encounter bazel certificate errors, you might want to set: BAZEL_CUSTOM_CACERTS=/path/to/trust.store (and optionally BAZEL_CUSTOM_CACERTS_PASSWORD), and to enable vcpkg to actually use it: VCPKG_KEEP_ENV_VARS=BAZEL_CUSTOM_CACERTS;BAZEL_CUSTOM_CACERTS_PASSWORD")
+			if(CMAKE_HOST_WIN32)
+				message(STATUS "(For BAZEL_CUSTOM_CACERTS please use forward slashes instead of backslashes on Windows systems.")
+			endif()
+		endif()
+	endif()
+
+	message(STATUS "Warning: Building TensorFlow can take an hour or more.")
+	set(COPTS "")
+	set(CXXOPTS "")
+	set(LINKOPTS "")
+	if(BUILD_TYPE STREQUAL dbg)
+		if(VCPKG_TARGET_IS_WINDOWS)
+			set(BUILD_OPTS "--compilation_mode=dbg --features=fastbuild") # link with /DEBUG:FASTLINK instead of /DEBUG:FULL to avoid .pdb >4GB error
+		else()
+			set(BUILD_OPTS "--compilation_mode=dbg")
+		endif()
+
+		separate_arguments(VCPKG_C_FLAGS)
+		separate_arguments(VCPKG_C_FLAGS_DEBUG)
+		foreach(OPT IN LISTS VCPKG_C_FLAGS VCPKG_C_FLAGS_DEBUG)
+			list(APPEND COPTS "--copt=${OPT}")
+		endforeach()
+		separate_arguments(VCPKG_CXX_FLAGS)
+		separate_arguments(VCPKG_CXX_FLAGS_DEBUG)
+		foreach(OPT IN LISTS VCPKG_CXX_FLAGS VCPKG_CXX_FLAGS_DEBUG)
+			list(APPEND CXXOPTS "--copt=${OPT}")
+		endforeach()
+		separate_arguments(VCPKG_LINKER_FLAGS)
+		separate_arguments(VCPKG_LINKER_FLAGS_DEBUG)
+		foreach(OPT IN LISTS VCPKG_LINKER_FLAGS VCPKG_LINKER_FLAGS_DEBUG)
+			list(APPEND LINKOPTS "--linkopt=${OPT}")
+		endforeach()
+	else()
+		set(BUILD_OPTS "--compilation_mode=opt")
+
+		separate_arguments(VCPKG_C_FLAGS)
+		separate_arguments(VCPKG_C_FLAGS_RELEASE)
+		foreach(OPT IN LISTS VCPKG_C_FLAGS VCPKG_C_FLAGS_RELEASE)
+			list(APPEND COPTS "--copt=${OPT}")
+		endforeach()
+		separate_arguments(VCPKG_CXX_FLAGS)
+		separate_arguments(VCPKG_CXX_FLAGS_RELEASE)
+		foreach(OPT IN LISTS VCPKG_CXX_FLAGS VCPKG_CXX_FLAGS_RELEASE)
+			list(APPEND CXXOPTS "--copt=${OPT}")
+		endforeach()
+		separate_arguments(VCPKG_LINKER_FLAGS)
+		separate_arguments(VCPKG_LINKER_FLAGS_RELEASE)
+		foreach(OPT IN LISTS VCPKG_LINKER_FLAGS VCPKG_LINKER_FLAGS_RELEASE)
+			list(APPEND LINKOPTS "--linkopt=${OPT}")
+		endforeach()
+	endif()
+
+	if(VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
+		if(VCPKG_TARGET_IS_WINDOWS)
+			list(JOIN COPTS " " COPTS)
+			list(JOIN CXXOPTS " " CXXOPTS)
+			list(JOIN LINKOPTS " " LINKOPTS)
+			vcpkg_execute_build_process(
+				COMMAND ${BASH} --noprofile --norc -c "${BAZEL} build --verbose_failures ${BUILD_OPTS} ${COPTS} ${CXXOPTS} ${LINKOPTS} --python_path='${PYTHON3}' --define=no_tensorflow_py_deps=true ///tensorflow:tensorflow_cc.dll ///tensorflow:install_headers"
```

```suggestion
				COMMAND ${BASH} --noprofile --norc -c "'${BAZEL}' build --verbose_failures ${BUILD_OPTS} ${COPTS} ${CXXOPTS} ${LINKOPTS} --python_path='${PYTHON3}' --define=no_tensorflow_py_deps=true ///tensorflow:tensorflow_cc.dll ///tensorflow:install_headers"
```

in case bazel is on a path with spaces?